pages/rewards/index.js

```diff
@@ -114,6 +117,7 @@ export function DonateButton () {
           <Input
             label='amount'
             name='amount'
+            type='number'
```

Comment:
This wasn't an issue before (meaning you could type other numbers beyond 1000), but the invoicing code was failing due to type validation. Of course, it is an issue allowing alpha chars haha